### PR TITLE
feat(scim): update okta scim provisioning docs

### PR DESCRIPTION
**Pull request by cathyteng17**
Add some more troubleshooting information and images relating to the UI changes.

### CODE REVIEW

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 1]
```diff
@@ -117,11 +127,21 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
 
 </Alert>
 
+<Alert level="warning">
```
Comment (JoshFerge) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:130]: i wonder if this should be alert level  "note", as how warning looks in the UI almost says to me that this is a bad thing

**Review by JoshFerge — APPROVED** [review]
looks good other than nit about alert level

**Review by lizokm — COMMENTED** [review]
Hey there! I left a bunch of suggested changes. Let me know if you'd like to talk through anything :)

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 2]
```diff
@@ -137,6 +157,22 @@ In order to sync a group in Okta as a team in Sentry:
...
+
+#### I'm unable to add or remove a member from my team in Sentry
+
+Groups in Okta that have been pushed to Sentry will create teams that have their team membership locked. Adding or removing members from these teams must be done through Okta.
+
+#### I removed a user from the Sentry app in Okta, but the user still exists in my Sentry organization
+
+Make sure that `Deactivate User` is enabled in the "Provisioning" tab of your Sentry application in Okta. Re-provision the user to your Sentry organization via Okta, then remove them again.
+
+#### How do I unset the organization-level role for a user?
```
[the remark this answers is not in the document]
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:172]: If you've set the role of a user through Okta, then the user's role won't be changeable from within the Sentry application. This is for if you want to remove the role set through Okta -- let me reword it as such 😅

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 3]
```diff
@@ -137,6 +157,22 @@ In order to sync a group in Okta as a team in Sentry:
...
+
+If the user is re-provisioned with a blank (default) role, then the user's role in Sentry won't be restricted via Okta.
+
```
[the remark this answers is not in the document]
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:175]: I will reword this and push again!

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 4]
```diff
@@ -117,11 +127,21 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
...
+![Okta Role Restricted Role Select](./okta-role-restricted-sentry-role-select.png)
+
 ### Troubleshooting
 
 #### I get an error in Okta when provisioning a user saying "Matching user not found"
 
-Make sure that `Create User` and `Deactivate User` are enabled in the "Provisioning" tab of your Sentry application in Okta
+Make sure that `Create User` and `Deactivate User` are enabled in the "Provisioning" tab of your Sentry application in Okta.
 
 #### I pushed a group to Sentry but the team is empty. (Or) I assigned Sentry to a group but no team was created?
```
[the remark this answers is not in the document]
Comment (cathyteng17) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx]: I separated it into two questions!

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx (hunk 3 above)
[the remark this answers is not in the document]
Comment (lizokm) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx]: Awesome, thanks!

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 5]
```diff
@@ -65,6 +65,16 @@ As a result of these changes, users who are assigned will be sent an invitation
 
 You can use "Push Groups" to sync and assign groups in Okta; they will be reflected in Sentry teams.
 
+<Alert level="note">
+
+If you use Okta to assign organization membership, you’ll be unable to make membership changes through Sentry and will need to continue using Okta.
```
Comment (leedongwei) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:70]: For users that are provisioned to Sentry through Okta, you'll be unable to make membership changes to them through Sentry. To remove these users, you will have to deprovision them in Okta and sync it to Sentry.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 6]
```diff
@@ -65,6 +65,16 @@ As a result of these changes, users who are assigned will be sent an invitation
...
+
+</Alert>
+
+![Okta Provisioned User Unable to Leave](./okta-provisioned-unable-to-leave.png)
```
Comment (leedongwei) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:74]: Remove vs Leave is the same thing, just different copy. You can get away with just showing "Leave".

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 7]
```diff
@@ -117,25 +127,49 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
 
 </Alert>
 
-### Troubleshooting
+<Alert level="note">
+
+If you use Okta to assign organization-level roles, you’ll be unable to make membership changes through Sentry and will need to continue using Okta.
```
Comment (leedongwei) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:132]: For users that are assigned a Sentry organization-level role through Okta, you'll be unable to make role changes to them through Sentry. You will have to edit the role in their app user profile in Okta and sync them to Sentry.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx [hunk 8]
```diff
@@ -117,25 +127,49 @@ If a user is a member of multiple Okta groups, only the attributes set by the fi
...
+
+Users who've had their roles assigned via Okta will only be able to make membership changes via Okta.
+
+![Okta Role Restricted Role Select](./okta-role-restricted-sentry-role-select.png)
+
+## Troubleshooting
+
+### I get an error in Okta when provisioning a user saying "Matching user not found".
+
+Make sure that `Create User` and `Deactivate User` are enabled in the "Provisioning" tab of your Sentry application in Okta.
```
Comment (leedongwei) [inline src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:144]: You should add a screenshot for this

**Review by leedongwei — APPROVED** [review]
Small nits but overall LGTM.

File: src/docs/product/accounts/sso/okta-sso/okta-scim.mdx (hunk 7 above)
Comment (AniketDas-Tekky) [reply src/docs/product/accounts/sso/okta-sso/okta-scim.mdx:132]: syncing doesn't do anything at the moment. If they absolutely need to change roles, they will need to deprovision and reprovision the user.

**Pull request merged**